src/client.rs

```diff
@@ -348,6 +349,12 @@ impl Client {
             .await;
     }
 
+    pub async fn read_messages(&self, room_id: OwnedRoomId) {
+        let messages_request = MessagesRequest::from_start(&room_id);
+        self.send_and_notify(messages_request, UserRequest::Messages)
```

Comment:
Would it make sense to track the total number of messages read per user? Thinking of having some kind of metric to compare against the current prd state.

Reply:
Number of messages read for each room?

Reply:
Exactly.

Reply:
Are we tracking that on kernel? I think we can deduce it from server side metrics (requests and room sizes).

Also we are requesting messages from the beginning (using from `0` filter) and I believe that's not the case in the explorer side (I saw those requests having some timestamp), not sure how should we act there...

Reply:
Ok, got it!